server.js

```diff
@@ -12,8 +17,94 @@ app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-app.use(express.static(path.resolve(__dirname, './build')));
+app.use(express.static(path.resolve(__dirname, './public')));
 app.use('/api/v1', router);
-app.get('*', (req, res) => res.sendFile(path.resolve(__dirname, './build', 'index.html')));
+app.get('/*', (req, res) => res.sendFile(path.resolve(__dirname, './public', 'index.html')));
```

Comment:
So this was being a little problematic. You're doing a get and basically it seems like its intercepting all of the gets and just rendering a your index.html